### ISSUE TITLE
The README needs cleaning up (and perhaps I can do it . .)

### ISSUE THREAD

**Issue by LinuxOnTheDesktop** [pdfpc/latex-pdfpc#4]
There are some errors of English in the README file. I can fix those via a PR. But something else that needs fixing is this:

![image](https://user-images.githubusercontent.com/47056543/176797449-e0324384-c023-45a8-8604-8bbff93ed0d6.png)

The link leads to https://github.com/pdfpc/pdfpc - where I cannot find any relevant documentation. If you tell me where the link should point, then I can make the link point to that location.

Also - though perhaps the relevant documentation explains this - just what does building latex-pdfpc do? Does it put some executable(s) somewhere on one's system? I need to know the answer to that too - if I am to clean up the README.

**Comment by AndreasBilke** [comment]
I thank you for pointing to this. It should link to the pdfpc latex package documentation. I compiled version can be found at ctan: https://ctan.org/pkg/pdfpc

**Comment by fnevgeny** [comment]
@AndreasBilke I think the original comment was about [this README](https://github.com/pdfpc/latex-pdfpc/blob/master/README.md). Perhaps the link should go to https://pdfpc.github.io/

**Comment by AndreasBilke** [comment]
But https://pdfpc.github.io/ has no documentation about how to use the LaTeX package right?

**Comment by fnevgeny** [comment]
Right, I was confused. So pdfpc/pdfpc-doc.tex should be the ultimate documentation.

**Comment by AndreasBilke** [comment]
And a compiled version of it sits at https://ctan.org/pkg/pdfpc :)

**Comment by fnevgeny** [comment]
But [README](https://github.com/pdfpc/latex-pdfpc/blob/master/README.md) still points to github :)

**Comment by AndreasBilke** [comment]
IMHO *the* LaTeX package documentation resides at https://mirrors.ctan.org/macros/latex/contrib/pdfpc/pdfpc-doc.pdf since this is the compiled version of the latest published pdfpc package at CTAN.

**Comment by LinuxOnTheDesktop** [comment]
Here is what I have come up with for the readMe file. I am hampered here by . . not actually know how any of this Latex stuff works! (So perhaps I should never have got into this. But anyway . .)

```
# latex-pdfpc

`latex-pdfpx` allows the pdf presenter console (`pdfpc`) to handle pdf files that contain LaTeX code.

## Dependencies

`latex-pdfpc` requires these packages:
[`kvoptions`](https://ctan.org/pkg/kvoptions);
[`xstring`](https://ctan.org/pkg/xstring);
[`iftex`](https://ctan.org/pkg/iftex);
[`hyperxmp`](https://ctan.org/pkg/hyperxmp). Download the packages from the links. The packages themselves contain information about how one is to install the packages.

If one wants pdfpc to make use, not just of LaTeX but also of *'LuaTeX'*, then two further packages are needed:
[`stringenc`](https://ctan.org/pkg/stringenc);
[`pdftexcmds`](https://ctan.org/pkg/pdftexcmds).

## Usage

For information on using `pdfpc` with `latex-pdfpx`, see [`this webpage`](https://mirrors.ctan.org/macros/latex/contrib/pdfpc/pdfpc-doc.pdf).
```

**Comment by fnevgeny** [comment]
> `latex-pdfpx` allows the pdf presenter console (`pdfpc`) to handle pdf files that contain LaTeX code.

It sounds like you don't actually use the package (and hence don't understand what it does)... Why do you bother "fixing" it then?

**Comment by LinuxOnTheDesktop** [comment]
I wanted to contribute and, having looked at the document too superficially, I reached this erroneous conclusion: all that was needed was to fix a link or two and to improve the English. Still, perhaps what I have done is worthwhile. For, perhaps, all we need now is a change or two by some person(s) who knows how the package works. If I am given such changes (to the text that I have proposed) then I can incorporate them into a PR.
